grafana_client/elements/team.py

```diff
@@ -215,6 +215,6 @@ def remove_team_external_group(self, team_id, group_id):
         :param group_id:
         :return:
         """
-        team_group_path = "/teams/%s/groups/%s" % (team_id, group_id)
+        team_group_path = "/teams/%s/groups?groupId=%s" % (team_id, group_id)
```

Comment:
Could it be something which changed only on more recent versions of Grafana? We would like to make sure not to break older versions in this regard. Is there a way to assess that?

Reply:
Yes, you are right. Thanks to notice that.

In the documentation [10.1](https://grafana.com/docs/grafana/v10.1/developers/http_api/team_sync/#remove-external-group)

`DELETE /api/teams/1/groups/cn=editors,ou=groups,dc=grafana,dc=org HTTP/1.1`

In the documentation [10.2](https://grafana.com/docs/grafana/v10.2/developers/http_api/team_sync/#remove-external-group)

`DELETE /api/teams/1/groups?groupId=cn%3Deditors%2Cou%3Dgroups%2Cdc%3Dgrafana%2Cdc%3Dorg HTTP/1.1`

I find nothing in the release notes. This API is only available for grafana enterprise. I open a case to grafana to check ...

Reply:
I copy/paste the support comments.

> It looks like this change to the Team Sync API was made back in Oct. 2023: https://github.com/grafana/grafana/pull/76325
>  
> This change was made because there was an issue on versions prior to 10.2 where external teams that were deleted would remain in the team_group table in the DB.

Reply:
Excellent. Thank you very much for clarifying and implementing, @hamelg.